.circleci/config.yml

```diff
@@ -85,53 +85,6 @@ jobs:
             fi
           no_output_timeout: 45m
 
-  slither:
```

Comment:
Why did we delete the tests?

Reply:
@draganm It seems that you just needed to upgrade the solidity version that slither uses instead of deleting all the tests.